startup_heroku.sh

```diff
@@ -29,5 +30,6 @@ export SMART_ANSWER_TO_OPEN="marriage-abroad"
 open $HEROKU_URL$SMART_ANSWER_TO_OPEN
 
 echo
+echo "*Set ERRBIT_API_KEY and ERRBIT_HOST manually to enable error reporting*"
 echo "All done"
```

Comment:
Should this tell you to set `ERRBIT_ENV` as well as `ERRBIT_API_KEY` and `ERRBIT_HOST`?